contrib/intel/jenkins/summary.py

```diff
@@ -941,8 +943,10 @@ def summarize_items(summary_item, logger, log_dir, mode):
     else:
         output_name = f'summary_{job_name}.log'
 
-
     full_file_name = f'{log_dir}/{output_name}'
+    if (not os.path.exists(full_file_name)):
+        with open(full_file_name, 'x') as output_file:
+            print(f"Writing summary to {full_file_name}")
```

Comment:
Python open() with mode 'a' is supposed to create the file if it doesn't exist so this part is redundant.

Reply:
fixed, thanks!